### PR TITLE
[12.0][MIG] account_global_discount: Migration to 12.0

### PR DESCRIPTION
**Pull request by manuelcalerosolis**
This is the migration of module account_global_discount

### CODE REVIEW

File: account_global_discount/__manifest__.py [hunk 1]
```diff
@@ -1,8 +1,9 @@
 # Copyright 2019 Tecnativa S.L. - David Vidal
+# Copyright 2020 Xtendoo - Manuel Calero
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 {
     'name': 'Account Global Discount',
-    'version': '11.0.1.1.0',
+    'version': '12.0.1.1.0',
```
Comment (chienandalu) [inline account_global_discount/__manifest__.py]:
```suggestion
    'version': '12.0.1.0.0',
```

File: account_global_discount/__manifest__.py [hunk 2]
```diff
@@ -1,8 +1,9 @@
 # Copyright 2019 Tecnativa S.L. - David Vidal
+# Copyright 2020 Xtendoo - Manuel Calero
```
Comment (chienandalu) [inline account_global_discount/__manifest__.py]: Please, just as contributor
```suggestion
```

**Review by chienandalu — CHANGES_REQUESTED** [review]
Looks like runbot has some warnings

**Comment by manuelcalerosolis** [comment]
@pedrobaeza milestone must be 12.0

File: account_global_discount/models/account_invoice.py [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+        readonly=True,
+    )
+    discount = fields.Float(
+        string='Discount',
+        readonly=True,
+    )
+    discount_display = fields.Char(
+        compute='_compute_discount_display',
+        readonly=True,
+        string="Discount",
```
Comment (chienandalu) [inline account_global_discount/models/account_invoice.py]: Just change this label to get rid of the runbot warning

Comment (pedrobaeza) [reply account_global_discount/models/account_invoice.py]:
```suggestion
        string="Displayed Discount",
```

**Comment by pedrobaeza** [comment]
Please avoid merge commits in the PR and there are a lot of commits for translation to German from same person that should be squashed.

We are going to need this module in some weeks, so if you want, we can supersed this.

**Comment by manuelcalerosolis** [comment]
ok delete it without problems.

**Comment by pedrobaeza** [comment]
Supersed by #748

**Pull request closed without merging**